app/presenters/buyers/accounts_index_presenter.rb

```diff
@@ -14,7 +14,7 @@ def initialize(provider:, params: {})
 
   def buyers
     @buyers ||= provider.buyer_accounts
-                        .includes([:admin_user])
+                        .includes(%i[admin_user])
```

Comment:
Everything looks awesome now. Just a nitpick. I prefer such minor style changes to stay out because they make git blame more complicated. I don't know whether you use some special tool to help with that.